content/logging/cleanup.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Cleanup Logging"
+date: 2018-08-07T08:30:11-07:00
+weight: 50
+draft: true
```

Comment:
I think you need to change 'draft: false' in order to show these pages in PR. Should be okay though because these PR's will be merged into 'onug' branch. @brentley  can you confirm?